docs/sitemap.txt

```diff
@@ -49,6 +49,7 @@ index.md
 			Windows-module.md
 			Cuda-module.md
 			Kconfig-module.md
+			Freedesktop-module.md
```

Comment:
I think it needs to be added in `docs/theme/extra/templates/navbar_links.html` too.